pkgs/by-name/po/polybar-pulseaudio-control/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, bash
+, coreutils
+, fetchFromGitHub
+, gawk
+, makeWrapper
+, pulseaudio
+, stdenv
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "polybar-pulseaudio-control";
+  version = "3.1.1";
+
+  src = fetchFromGitHub {
+    owner = "marioortizmanero";
+    repo = "polybar-pulseaudio-control";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-egCBCnhnmHHKFeDkpaF9Upv/oZ0K3XGyutnp4slq9Vc=";
+  };
+
+  dontConfigure = true;
+  dontBuild = true;
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mv pulseaudio-control.bash pulseaudio-control
+    install -Dm755 pulseaudio-control -t $out/bin
+    wrapProgram "$out/bin/pulseaudio-control" \
+      --prefix PATH : "$out/bin:${lib.makeBinPath [ bash coreutils gawk pulseaudio ]}"
```

The program doesn't seem to execute itself from `$PATH`. It would be problematic anyway, since the binary has been renamed. Let's remove `$out/bin` from `$PATH`

```suggestion
      --prefix PATH : "${lib.makeBinPath [ bash coreutils gawk pulseaudio ]}"
```